addons/maaacks_game_template/extras/scripts/LevelLoader.gd

```diff
@@ -23,7 +23,7 @@ func get_level_file(level_id : int = get_current_level_id()):
 	if files.is_empty():
 		push_error("files is empty")
 		return
-	if level_id >= files.size():
+	if level_id > files.size():
 		push_error("level_id is greater than files size")
```

Comment:
Since `level_id` is zero-indexed, `level_id` shouldn't equal `files.size()`. In case `files.size() = 1`, then `level_id = 0`, so the statement would be:

`if 0 >= 1: push_error...`

I think throwing an error is intended. Let me know if that doesn't make sense here for some reason, though.

We could instead update the wording of the error to read that the "level_id is out of bounds of the levels list" so that it's more accurate.

Reply:
Came back to this with fresh eyes and did some more digging - found that the current level is saved into config at some point, then when you try autoload the levels based on the array, it gives this error when the saved-in-config current_level is larger than the auto-generated max (which looked to me to be getting pulled from a directory listing?)

I'll close this and raise an issue for it, sorry about the mix-up